tests/test_integration.py

```diff
@@ -92,13 +96,15 @@ def test_connect(self):
 
 
 class CommandTest(TestCase):
+    def setUp(self):
+        node.reset()
 
     def _output_format(self, format, func, query="select name from sys.cluster"):
         orig_argv = sys.argv[:]
         try:
             sys.argv = ["testcrash",
                         "-c", query,
-                        "--hosts", node.http_url,
+                        "--hosts", node.get_http_url(),
```

Comment:
Ah, now I see the reason why you introduced `http_url` as a _property_ to the framework-agnostic test adapter. Apologies for not following about the rationale right away.

Now, as we can see the consequences this would have to a downstream code base, my suggestion would be to re-add the `http_url` property there for compatibility reasons. I think it will do no harm to have it, additionally to the canonical `get_http_url()` method.

Reply:
Fixed

Reply:
Thank you!